### ISSUE TITLE
easy access to pretrained models

### ISSUE THREAD

**Issue by denizyuret** [ai-ku/Morse.jl#6]
To use pretrained models should be something as simple as: `MorseModel(url)` and some standard predict interface. The list of valid URLs and their properties (language, training data, test accuracy etc) should be documented.

**Comment by denizyuret** [comment]
@ekinakyurek where are we with this? 
* Do we have all the models in all the languages saved somewhere or do we need training? 
* If we have saved models do they work with current code? 
* In what format do we / should we save them for public use?
* Where should we save them for public use?
* What API should we use to download/use them?
